AVR code/MySC1.1.c

```diff
@@ -1,12 +1,20 @@
+/*
+ * VOC_1.cpp
+ *
+ * Created: 21-01-2016 15:19:48
+ *  Author: Harsimar
+ */ 
 #define F_CPU 8000000UL
 #include <avr/io.h>
 #include <util/delay.h>
 #include <math.h>
 #include "notes.h"
-#include "USART_32.h"
-int flag = 0, i = 0, k = 0, max = 0;
+#include "USART_128.h"
+#include "EEPROM.h"
+int flag = 0, current_song_node = 0, k = 0, max = 0,song_no=0,tempo=0;
 int temp_array[10];
 
+
 void pwm_init()
```

Comment:
Change pwm_inti for ATmega128, i.e. DDRD |= _BV(PD4) should be changed to DDRB |= _BV(PB6)